src/components/Footer.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import styled from "styled-components";
+import { StaticImage } from "gatsby-plugin-image";
+
+const FooterHolder = styled.div`
+  margin-top: 5px;
+  width: 100%;
+  min-height: 50px;
+`;
+
+const SocialLogoHolder = styled.div`
+  display: flex;
+  justify-content: flex-end;
+  float: right;
+`;
+
+const Footer = () => {
+  return (
+    <FooterHolder>
+      <p style={{ float: "left", "margin-top": "25px" }}>
+        © {new Date().getFullYear()} York Community Web and App Development
+      </p>
+      <SocialLogoHolder>
+        <a href="https://facebook.com/yordevs" target="_blank" rel="noreferrer">
+          <StaticImage
```

Comment:
Can we use icons from the installed `react-icons` package instead of images? You can grab social media icons from the `react-icons/fa` package. [Link](https://react-icons.github.io/react-icons/icons?name=fa)